benchmarks/hrr_expand_ablation.py

```
@@ -0,0 +1,271 @@
+        score.expand_hits += _R._LAST_TELEMETRY.hrr_expand
+        prediction = _context(result.beliefs)
+        if qa.category == 5 and len(prediction.split()) < 10:
+            prediction = "No information available"
+        answer = qa.answer if qa.category != 5 else ""
+        score.add(qa.category, score_qa(prediction, answer, qa.category))
+
+
+def main() -> None:
+    ap = argparse.ArgumentParser(description=__doc__)
+    ap.add_argument("--data", default=DEFAULT_DATA_PATH)
+    ap.add_argument("--out", default="/tmp/hrr_expand_ablation.json")
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/hrr_expand_ablation.json"

(S108)